server/server/src/cold_chain/sensor.rs

```diff
@@ -151,7 +151,7 @@ fn upsert_sensor(
                 id: id.clone(),
                 name: Some(sensor.name.clone()),
                 is_active: None,
-                location_id: None,
+                location: None,
```

Comment:
wasn't `location_id`` the better name?

Reply:
I agree, location_id is better. This was changed and missed from a [this](https://github.com/openmsupply/open-msupply/pull/2363) PR..